sites/cheerpx/src/content/docs/12-reference/20-CheerpX-linux-callbacks.md

```diff
@@ -0,0 +1,72 @@
+---
+title: CheerpX.Linux event callbacks
+description: Event Callbacks for CPU and Disk Monitoring
+---
+
+The `CheerpX.Linux` instance returned by `create` provides methods to register and unregister callbacks to monitor CPU and disk activity, as well as disk latency.
+
+### `registerCallback`
+
+```ts
+registerCallback(eventName: string, callback: (state: string | number) => void): void
+```
+
+Registers a callback function for a specific event type.
+
+**Parameters**:
+
+- **eventName**: A string specifying the event type. Can be `"cpuActivity"`, `"diskActivity"`, or `"diskLatency"`.
+- **callback**: A function that will be called when the event occurs. It receives a parameter which varies based on the event type:
+  - For `"cpuActivity"` and `"diskActivity"`: `state` can be either "ready" (active) or "wait" (idle).
+  - For `"diskLatency"`: `state` is a number representing the time in milliseconds between requesting and receiving the last disk block.
+
+### `unregisterCallback`
+
+```ts
+unregisterCallback(eventName: string, callback: (state: string | number) => void): void
+```
+
+Unregisters a previously registered callback function for a specific event type.
+
+**Parameters**:
+
+- **eventName**: A string specifying the event type. See the `registerCallback` reference above for the supported event types.
+- **callback**: The function to be unregistered.
+
+Example usage:
+
+```js
+function hddCallback(state) {
+	var h = document.getElementById("hddactivity");
+	if (state == "ready") h.textContent = "\u{1F7E2}";
```

Comment:
This example code came from the legacy WebVM and reference Unicode chars. Please rewrite this to update the HTML textContent to a string that make sense.